db/migrate/20190531210309_limit_books_title.rb

```diff
@@ -0,0 +1,12 @@
+class LimitBooksTitle < ActiveRecord::Migration[5.1]
+  def up
+    Book.where('length(title) > 25').find_each do |book|
+      book.update title: book.title.first(22) + '...'
```

Comment:
:warning: NEVER use DML operations in migrations. Migrations only should modify schema (only DDL operations)
[Difference between DML and DDL](https://stackoverflow.com/questions/2578194/what-is-ddl-and-dml)